api/ai/priority_algorithm/priority/priority.py

```diff
@@ -221,6 +240,10 @@ def get_schema() -> Schema:
             }
         )
 
+    @staticmethod
+    def parse_json(data: Dict) -> "RequirementPriority":
+        return RequirementPriority()
```

Comment:
doesnt this have `criteria`?

Reply:
This one does not have criteria as attribute (although the schema said it needs it)

Reply:
This might mean either the schema or the class is outdated